content/guides/authentication/sso.md

```diff
@@ -21,13 +21,12 @@ being built.
 
 ## Custom Apps with Client-side Authentication
 
-When using an application configured as a [Custom App][custom_app] that uses
+When using a [Custom App][custom_app] that uses
 client-side [OAuth 2.0] to authenticate users, the Box API will automatically
-redirect a user's browser to the configured SSO log-in screen.
+redirect a user's browser to their enterprise's configured SSO log-in screen.
```

Comment:
So just to clarify, the user is redirected to their own enterprise's SSO screen. Right? External users might potentially not see any SSO screen if their org is configure differently.

Reply:
Correct, and agreed! Added some brief context to handle this case:

"When users authenticate with a [Custom App][custom_app] configured to use
client-side [OAuth 2.0], Box's OAuth2 grant page will detect if a user's
enterprise is configured to use SSO. If so, Box will redirect the user's browser
to their own enterprise's configured SSO log-in screen."